src/main/java/seedu/address/model/Scheme/Fasscheme.java

```diff
@@ -0,0 +1,27 @@
+package seedu.address.model.scheme;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents a MOE FAS scheme in the address book.
+ */
+public class Fasscheme extends Scheme {
```

Comment:
The class should be named as `FasScheme` as per coding standard

Reply:
Should we use `FinancialAssistanceScheme` instead?

Reply:
Second this. Spelling out the full name is clearer.

Reply:
Thanks for pointing it out! I feel that some schemes names can be quite similar and long which is why I did it in acronym style, thoughts?

Reply:
I think it's better to use the full name since we are not familiar with the different schemes in Singapore.

Reply:
Yup have changed it to reflect full name of the schemes, thanks for the input!